src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfDuplexService.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+
+namespace WcfService
+{
+    [ServiceBehavior(AddressFilterMode = AddressFilterMode.Any)]  
+    public class WcfDuplexService : IWcfDuplexService, IWcfDuplexService_DataContract, IWcfDuplexService_Xml
```

Comment:
I maintain the same folder structure as the current WCF service, just replace endpoint resource with testhosts. We can discuss rearrange them later on.